### ISSUE TITLE
BGR_U8 Encoding doesn't work with WebGL

### ISSUE THREAD

**Issue by jleibs** [rerun-io/rerun#7454]
```
pixi run rerun-web
pixi run -e py python tests/python/release_checklist/check_bgr.py 
```

Browsers: Firefox and Chrome both show a black tile:
![image](https://github.com/user-attachments/assets/b8327476-3edc-43a2-8b1f-98bc4da67b6b)

Works fine on native.

OS: Arch, Wayland, Nvidia (560 drivers)

**Comment by jleibs** [comment]
Additional info from web console:
![image](https://github.com/user-attachments/assets/96c0fd68-b79a-4d20-b5f5-007959c9e768)

**Comment by Wumpf** [comment]
Turns out webgl doesn't support BGR(A)
https://registry.khronos.org/OpenGL-Refpages/es2.0/xhtml/glTexImage2D.xml

I wonder how this fell through the cracks. I definitely tested Web when introducing this feature and I'm _fairly_ sure I had a look at WebGL _and_ WebGPU (where this works via spec). Might want to check if some WebGL impls allow this or (more likely) I only tested WebGPU.

Also there's a wgpu bug here where it doesn't set the downlevel flag for missing texture support.
